src/docs/release-notes/R22/R22.mdx

```diff
@@ -0,0 +1,56 @@
+### Brightlayer UI Migration
+
+The Power Xpert Blue "PX Blue" design system rebranded to Brightlayer User Interface "Brightlayer UI".
+
+-   [Brightlayer UI](https://brightlayer-ui.github.io/) documentation site
+-   Github organization migrated to [Brightlayer UI](https://github.com/brightlayer-ui)
+-   Component libraries migrated to [Brightlayer UI](https://brightlayer-ui-components.github.io/)
+-   NPM packages migrated to [Brightlayer UI](https://www.npmjs.com/org/brightlayerui)
+
+### Component Libraries
+
+-   Updated Angular Score Card component styles
+-   Updated React Info List Item with conditional title and subTitle rendering
+-   Added React Empty State card variation
+
+### Icons
+
+-   Added 38 new icons
+-   Fixed an icon clipping issue that caused wrong rendering in icon fonts
+
+### Documentation Site
+
+-   Added [migration instructions](https://brightlayer-ui.github.io/migration) for package migration
+-   Added new design patterns for Angular and React
+-   Updated design patterns to be responsive
+
+### Figma Component Sticker Sheet
+
+-   Added Multi-Line Text Field component and Progress Icons
+
+### Miscellaneous
+
+-   Bug fixes & minor updates
+
+### Package Summary
+
+Besides the package migrations to the new @brightlayer-ui handler, we also made some minor updates to the following packages.
```

Comment:
I agree with Tom, this blurb "Besides the package.... following packages" could be removed to match the formatting from last release.